packages/terra-application-docs/src/dev-site-extended-test/extended/extended.extended.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Do files in this folder need to have `.extended` as a naming convention in the file name? At first glance `extended/extended.extended.jsx` appears repetitive.

Reply:
Yeah it does, this is just for testing purposes, we could rename things but it doesn't really matter since it's just for testing.
The config that has us use all this extended stuff comes from here: https://github.com/cerner/terra-application/blob/terra-dev-site-v7/webpack.config.js#L33